### ISSUE TITLE
Multi Page Notes

### ISSUE THREAD

**Issue by Nebucatnetzer** [weirdNox/org-noter#15]
It would be nice to have a way to assign multiple pages to a  note.

**Comment by weirdNox** [comment]
Hey there! Thanks for the suggestion, that's something that was on the back of my head too. 

However, every time I think about this and how it would work, I can't think of a nice way to fit this in... We would need a command to add another page to the selected note, and a command to remove, and then we would need to update every command that is already implemented to, instead of checking just a page per note, to check a list of pages. That's ok, it would take time, but not impossible.

But is it worth it? What's the point after all? With things like note grouping, a parent note can be extended by having many notes inside it, covering different pages.

Besides, how would syncing by page work? Should it stop in every page that a given note covers? Or just some? 

Does this feature even make sense by assigning pages one by one? Maybe it would work out better as a range?

I don't know... What I've been thinking lately is that there could be an option that, when enabled, makes each note "active" until the next page with notes, like a continuous mode. This solves the problem with user interaction, but limits in flexibility.

**Comment by Nebucatnetzer** [comment]
I should've provided a bit more details at the time I opened this request.
I've already forgot why exactly I thought I would need this.
I just remember that I was taking notes on the slides from one of my classes and a topic span multiple slides which I wanted to summarise in one note. I think that was it but I have the feeling that I'm missing something. The slides came one after the other so it would be more a case of "page 2-8".

**Comment by weirdNox** [comment]
I'll tag this as an enhancement, but I will not implement this for the foreseeable future because I'm busy with college once again. If you have the time and want to try to do it yourself, go ahead! :)

**Comment by Nebucatnetzer** [comment]
Haha I reckon I'm in a similar position at the moment. I'm having currently the final exams of this semester :). Maybe I can look into it sometime during the summer. However I'm a really bad elisp coder. I still haven't quite figured out how it really works.

**Comment by weirdNox** [comment]
Hello! Finally had some time to look at this.

After analyzing the problem, I decided that I would do the kind of multi-page notes that are continuous, because those require _zero_ user interaction to create. What I implemented is a system that, when either you don't have any notes on a certain page, or all the notes in that page are below a certain "tipping point", then it will automatically also show the closest previous note.

An **example** use of this feature is to create a note at the beginning of a certain chapter, read until the middle, many pages after, without taking notes, and then taking a note without filing it under its own heading, but instead under the main chapter note. While that note is really only at the beginning, without things in the middle, it will also show up after.

This is something I have wanted for a very long time and may be disabled if it doesn't serve your purposes or is too confusing :)

The variable to look at is `org-noter-closest-tipping-point`.
